### ISSUE TITLE
Filter out process events based on CWD

### ISSUE THREAD

**Issue by Y4nko** [osquery/osquery#8043]
Hi all,

I would like to know if it would be possible to filter out process events in Linux using CWD, current working directory?

I couldn't find in the documentation the available options and wanted to know if the filtering is only limited to path or cmdlike parameters.

Thanks,

**Comment by michael-myers** [comment]
Yes, you can filter process events by CWD using the `cwd` column. Filtering is not limited to `path` or `cmdline`.

Example:

```
SELECT pid, path, cmdline, cwd
FROM process_events
WHERE cwd = '/opt/myapp';
```

But the `WHERE` clause filtering in SQL is applied after the events are received from the kernel and buffered. So if you're attempting a performance optimization then it may not achieve that, but if you're just interested in filtering the results it will work.
